api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -63,8 +64,8 @@
 
 import java.io.Closeable;
 import java.io.IOException;
-import java.util.Date;
```

Comment:
nitpick: this just got moved one line below, for no good reason

Reply:
My IDE is grouping dependencies by package. If not line 66, it would be 67.

Reply:
Well, at least it was alphabetically sorted within the `java.util` group ;)